docs/en/config_fw/gain_compression.md

```diff
@@ -0,0 +1,18 @@
+# Gain compression
+
```

Comment:
What version will this go into? v1.17, v1.18?

Assuming v1.17 you would add this

```suggestion
<Badge type="tip" text="main (planned for: PX4 v1.17)" />

```

Can we have an addition to the release note too?